wp-cli/vip-migrations.php

```diff
@@ -20,7 +20,15 @@ class VIP_Go_Migrations_Command extends WPCOM_VIP_CLI_Command {
 	 */
 	function dbdelta( $args, $assoc_args ) {
 		global $wpdb;
-
+		
+		$tables =  $args[1];
+		
+		if ( empty( $tables ) ){
+			$tables = '';
+		} else if ( ! in_array( $args[1], array( '', 'all', 'blog', 'global', 'ms_global' ), true ) ) {
+			WP_CLI::error( sprintf( '%s is not a valid entry. Valid entries are: "all", "blog", "global", "ms_global", "" (blank string)', $tables ) );
```

Comment:
If we move the list of valid values into a separate variable (array), we can just append that to the error message instead of hard-coding the values twice (something like `sprintf( '...Valid entries are one of: "%s"', implode( '","', $valid_tables ) );`